lib/tarfiles.py

```diff
@@ -70,6 +73,9 @@ def do_tarballs(args):
     we convert the argument to the next type as we go...
     """
 
+    NUM_RETRIES_UPLOAD = 20
```

Comment:
I'd like to see some of these configurable. Env vars maybe?

Reply:
Or you can make a case why they don't need to be configurable.

Reply:
Done.  These are now configurable via env vars (I'll need to document these when I write the jobsub_lite documentation.